apps/freelance/src/pages/FindJobs/index.tsx

```diff
@@ -1,60 +1,101 @@
 import { useState } from 'react';
-import { Button, List } from 'antd';
+import { Button, Input, List, PaginationProps } from 'antd';
 import { useTranslation } from 'react-i18next';
 import { FilterFormItems, Filters, JobCard } from '@freelance/components';
+import { useFindJobsQuery } from 'redux/services/jobsApi';
+import { GetJobParams } from 'redux/types/jobs.types';
 
-import { filterRight, filterTop, jobsList } from './constants';
-import { PageBar, StyledPagination, TitleContainer, Wrapper } from './styles';
+import { fetchLimit, filterRight, filterTop } from './constants';
+import {
+  PageBar,
+  PageBarRightSideContainer,
+  StyledPagination,
+  TitleContainer,
+  Wrapper,
+} from './styles';
+import { getFilterParams } from './utils';
 
 const FindJobs = () => {
   const [filtersVisibility, setFiltersVisibility] = useState<boolean>(false);
+  const [offset, setOffset] = useState<number>(0);
+  const [filterPayload, setFilterPayload] = useState<GetJobParams>();
+  const [search, setSearch] = useState<string>();
   const { t } = useTranslation();
+  const { Search } = Input;
```

Comment:
this initialise with each rerender, we dont need to have it inside the component

Reply:
@lyudad fixed